keyboards/xd75/keymaps/Revok75/keymap.c

```diff
@@ -0,0 +1,111 @@
+   { KC_CAPS,  KC_A,     KC_S,     KC_D,     		KC_F,     	KC_G, 	KC_H,   KC_J,   				KC_K,     KC_L,     KC_SCLN,  KC_QUOT,     LCTL(KC_R), KC_NO,       KC_ENT},
+   { KC_LSFT,  KC_Z,     KC_X,     KC_C,     		KC_V,     	KC_B, 	KC_N,   KC_M,   				KC_COMM,  KC_DOT,   KC_SLSH,  KC_BSLS,     KC_RSFT,    KC_UP,       LCTL(KC_S)},
+   { KC_LCTL,  KC_LALT,  KC_LGUI,  LCTL(KC_F), 	LCTL(KC_T), KC_SPC, KC_NO,	LCTL(KC_GESC),	MO(_FN), 	KC_RGUI,  KC_RALT,  KC_RCTL,     KC_LEFT,    KC_DOWN,     KC_RGHT},
+  },
+
+/* FUNCTION
+* .--------------------------------------------------------------------------------------------------------------------------------------.
+* | ESC    | F1     | F2     | F3     | F4     | F5     | F6    | F7     | F8     | F9     | F10    | -       | MUTE  | VOLDN   | VOLUP  |
+* |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+* | TAB    | -      | WIN    | -      | -      | RGB_TOG| -     | -      | OPTION  | RESET  | -      | [       | ]     | -       | -      |
+* |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+* | CAP LK | MAC    | RAINBOW| PLAIN  | -      | -      | -     | -      | -      | -      | ;      | '       | ENTER |  ENTER  | REF    |
+* |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+* | LSHIFT | INC    | DEC    | HUE I  | HUE D  | SAT I  | SAT D | -      | -      | .      | /       | \      | RSHIFT|  HOME   | SAVE   |
+* |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+* | LALT   | LGUI   | LALT   |  FIND  | CMD T	 | SPACE | SPACE  | ALFRED | FN-MO  | RALT   | RGUI    | RCTRL  | PGUP   | END    | PGDN   |
+* '--------------------------------------------------------------------------------------------------------------------------------------'
+*/
+
+ [_FN] = { /* OSLAYOUT + NUMPAD + MEDIA + LIGHTING */
+   { KC_TRNS, KC_F1,     KC_F2,						 KC_F3,    		   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_TRNS, KC_MUTE, KC_VOLD, KC_VOLU},
+   { KC_TRNS, KC_NO,     DF(_QW_W),    		 KC_NO,    		   KC_NO,   RGB_TOG, KC_NO,   KC_NO,   KC_RALT, RESET,   KC_NO,   KC_LBRC, KC_RBRC, KC_TRNS, KC_TRNS},
+   { KC_TRNS, DF(_QW_M), RGB_MODE_RAINBOW, RGB_MODE_PLAIN, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS},
+   { KC_TRNS, RGB_VAI,   RGB_VAD,  				 RGB_HUI,  		   RGB_HUD, RGB_SAI, RGB_SAD, KC_NO,   KC_NO,   KC_DOT,  KC_TRNS, KC_TRNS, KC_TRNS, KC_HOME, KC_TRNS},
+   { KC_TRNS, KC_TRNS,   KC_TRNS,  				 KC_TRNS,  		   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PGUP, KC_END,	 KC_PGDN},
+ }
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Again, everything from Line 92 to the end of the file can be deleted.

Reply:
resolved